setup.py

```diff
@@ -40,6 +40,7 @@
     ],
     entry_points={
         'console_scripts': [
+            'krux-s3-test = krux_s3.cli:main',
```

Comment:
It goes something like this: `'CLI_program_name = module(directory).module(sub-directory).module(file):function'`